### PR TITLE
feat: add windows cmd & powershell init scripts

### PR DESCRIPTION
**Pull request by rafaelramalho19**
Closes https://github.com/ipfs-shipyard/ipfs-webui/issues/1008

Screenshot of the end result:
image

### Notes

Also changed the "Start the IPFS daemon" text to match one common use case:
- User has already started daemon, but didn't setup the headers. After setting up the headers, the user is confused why it's not working. Maybe reading the "restart daemon" will help.

### CODE REVIEW

File: src/welcome/WelcomePage.js [hunk 1]
```diff
@@ -72,14 +72,22 @@ const ConnectionStatus = ({ t, connected, sameOrigin }) => {
...
+          </Shell>
+          <Shell title="Windows CMD" className="mt4">
+            <code className='db'><b className='no-select'>$ </b>ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin "[{addOrigin && `"""${origin}""", `}"""{defaultDomains.join('""", """')}"""]"</code>
+            <code className='db'><b className='no-select'>$ </b>ipfs config --json API.HTTPHeaders.Access-Control-Allow-Methods "["""PUT""", """POST"""]"</code>
+          </Shell>
         </div>
       )}
       <Trans i18nKey='notConnected.paragraph3'>
-        <p>Start an IPFS daemon in a terminal:</p>
+        <p>Start/Restart the IPFS daemon in a terminal:</p>
```
Comment (jessicaschilling) [inline src/welcome/WelcomePage.js:90]: Maybe “Then, start/restart the...” to make it clearer this is the second step (and lowercase “restart”.

**Comment by jessicaschilling** [comment]
This is great 😊 Just one tiny text suggestion. Thanks!

File: src/welcome/WelcomePage.js (hunk 1 above)
Comment (rafaelramalho19) [reply src/welcome/WelcomePage.js:90]: Done :)

**Comment by rafaelramalho19** [comment]
After discussing with @jessicaschilling, we came with this simple way to further help the user differentiate between which OS bash commands to use: https://i.imgur.com/nJ8JRhL.png

**Comment by jessicaschilling** [comment]
Let's remove the Apple icon per: https://www.apple.com/legal/intellectual-property/guidelinesfor3rdparties.html

**Comment by rafaelramalho19** [comment]
Thanks for looking into that. I reverted the icon changes then 👍

**Pull request merged**